course_info/icommons.py

```diff
@@ -1,80 +1,94 @@
-#from urllib import urlencode
-#from urlparse import urlparse
-#from requests.utils import parse_header_links
-#from django.http import QueryDict
-#from getenv import env
 import drest
-from drest.exc import dRestRequestError, dRestError
+import json
+import logging
+
+from django.core.cache import cache
 from canvas_course_info.settings import aws as settings
 
-import logging
 log = logging.getLogger(__name__)
 
-API_PATH = '/api/course/v2/'
-MAX_ITEMS_PER_REQUEST = 100
-MAX_CONCURRENT_REQUESTS = 4
+CACHE_KEY_COURSE_BY_CANVAS_COURSE_ID = 'course-by-canvas-course-id-{}'
+CACHE_KEY_COURSE_BY_COURSE_INSTANCE_ID = 'course-by-course-instance-id-{}'
+CACHE_KEY_SCHOOL_BY_SCHOOL_ID = 'school-by-school-id-{}'
 
-class CourseUpdateError(dRestRequestError):
-    pass
 
 class ICommonsApi(drest.API):
-
     @classmethod
     def from_request(cls, request):
         icommons_base_url = settings.ICOMMONS_BASE_URL
         access_token = settings.ICOMMONS_API_TOKEN
         return cls(icommons_base_url, access_token)
 
-
     def __init__(self, icommons_base_url, access_token):
         self.icommons_base_url = icommons_base_url
```

Comment:
why bother storing either of these on the object if they're never used again outside of the `__init__` method?